wgpu/src/lib.rs

```diff
@@ -5143,6 +5145,20 @@ pub enum Error {
         /// Description of the validation error.
         description: String,
     },
+    /// Internal GPU error, used for signaling failures in the implementation of WebGPU or the
+    /// system
+    Internal {
+        /// Lower level source of the error.
+        #[cfg(send_sync)]
+        #[cfg_attr(docsrs, doc(cfg(all())))]
+        source: Box<dyn error::Error + Send + 'static>,
+        /// Lower level source of the error.
+        #[cfg(not(send_sync))]
+        #[cfg_attr(docsrs, doc(cfg(all())))]
+        source: Box<dyn error::Error + 'static>,
+        /// Description of the internal GPU error.
```

Comment:
```suggestion
        /// Description of the internal error.
```

`GPU` implies that the GPU had a problem

Reply:
Agreed. Pushed as a `fixup!` with 971f96cc4.